### PR TITLE
Move to new staging area

### PR DESCRIPTION
**Pull request by mikeminutillo**
The old staging location has been removed and I cannot recreate one with the same details as Azure prevents me from doing so.

### CODE REVIEW

**Comment by ramonsmits** [comment]
@mikeminutillo Why is this needed? No info on why the change is required.

**Comment by mikeminutillo** [comment]
@ramonsmits updated

File: src/PlatformInstaller.ClickOnce/PlatformInstaller.ClickOnce.csproj [hunk 1]
```diff
@@ -80,7 +80,7 @@
       <ProdDeployManifest>PlatformInstaller.application</ProdDeployManifest>
       <TestDeployManifest>Test-PlatformInstaller.application</TestDeployManifest>
       <ProdDeployUrl>https://s3.amazonaws.com/particular.downloads/PlatformInstaller/</ProdDeployUrl>
-      <TestDeployUrl>https://particulartest.blob.core.windows.net/platforminstaller/</TestDeployUrl>
+      <TestDeployUrl>https://platforminstallerstaging.blob.core.windows.net/platforminstaller/Test-ParticularPlatform.exe</TestDeployUrl>
```
Comment (andreasohlund) [inline src/PlatformInstaller.ClickOnce/PlatformInstaller.ClickOnce.csproj:83]: Should the actual exe be in there? (the Prod URL doesn't have it?)

Comment (mikeminutillo) [reply src/PlatformInstaller.ClickOnce/PlatformInstaller.ClickOnce.csproj:83]: Curse, you are right. I'll fix it.

**Pull request merged**